Packs/CommonScripts/Scripts/GenerateAsBuilt/GenerateAsBuilt.py

```diff
@@ -825,7 +886,7 @@ def get_playbook_dependencies(playbook_name):
 
 def get_custom_automations():
     r = post_api_request(DEMISTO_AUTOMATIONS_PATH, {"query": "system:F AND hidden:F"}).get("scripts")
-    rd = SortedTableData(r, "Custom Automations", "name")
+    rd = SortedTableData(r, "Custom Automatons", "name")
```

Comment:
Why entering a type here?

Reply:
i now see the spelling suggestion here is wring so i switched back to 'Automations'. Other than that i don't see whats wrong here.

Reply:
The typo is the only issue :)